hw/bsp/rp2040/family.cmake

```diff
@@ -161,14 +161,22 @@ if (NOT TARGET _rp2040_family_inclusion_marker)
 		rp2040_family_configure_example_warnings(${TARGET})
 	endfunction()
 
+	function(family_add_pico_pio_usb TARGET)
+		target_link_libraries(${TARGET} PUBLIC tinyusb_pico_pio_usb)
+	endfunction()
+
 	function(family_configure_host_example TARGET)
 		family_configure_target(${TARGET})
 		target_link_libraries(${TARGET} PUBLIC pico_stdlib tinyusb_host)
 		rp2040_family_configure_example_warnings(${TARGET})
-	endfunction()
 
-	function(family_add_pico_pio_usb TARGET)
-		target_link_libraries(${TARGET} PUBLIC tinyusb_pico_pio_usb)
+		# For rp2040 enable pico-pio-usb
+		if (TARGET tinyusb_pico_pio_usb)
+			# code does not compile with GCC 12+
```

Comment:
The comment doesn't match the line below.
Does it fail with gcc 12+ (as the comment suggests) or with 11.3+ as the line below and your explanations in this PR suggest it?

Reply:
ah, it fails with 11.3 too (I updated the line but not the comment)

Reply:
I plan to push a fix to pico-pio-usb anyway

Reply:
> I plan to push a fix to pico-pio-usb anyway

Just for cross-linking purposes: this happened in https://github.com/sekigon-gonnoc/Pico-PIO-USB/pull/70